opentelemetry-api/tests/trace/test_span_context.py

```diff
@@ -0,0 +1,32 @@
+# Copyright The OpenTelemetry Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import pickle
+import unittest
+
+from opentelemetry import trace
+
+
+class TestSpanContext(unittest.TestCase):
+    def test_span_context_pickle(self):
```

Comment:
could you add a quick docstring to describe why it's important for SpanContext to be pickleable? It's nice to have an understand of why the behavior is there, rather than just a test to make sure it is.

Reply:
Done